crypto/tests/test_check_cashaddr.h

```diff
@@ -55,9 +55,8 @@ START_TEST(test_cashaddr) {
     rawdata_len = base58_decode_check(valid_cashaddr[i].legacy, HASHER_SHA2D,
                                       rawdata, sizeof(rawdata));
     ck_assert_int_eq(rawdata_len, 21);
-    ck_assert_int_eq(prog[0], rawdata[0] == 0   ? 0x00
-                              : rawdata[0] == 5 ? 0x08
-                                                : -1);
+    ck_assert_int_eq(prog[0],
```

Comment:
These `.h` and `.c` files were probably formatted and added by accident?

Reply:
indeed, not sure why make style reformats it, I forgot to omit it when adding changes to the commit, will fix that